### DISCUSSION TITLE
What is the path to a public collection?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16087] [Archive]
I made a collection `Contact` with one field `Email`. That collection has access for the `Public` role.
But `admin/content/contact/+` always redirects to Directus login page.

**Comment by rijkvanzanten** [comment]
`/items/contact` would be the public API endpoint 👍🏻 

The admin app is an authenticated environment, so that doesn't have public viewable data like that (beyond manually shared items from the item detail page)

**Reply by Pachat** [reply]
Thanks.  `/items/contact` shows indeed the data.
But I thought that I could access publicly the form I see as admin.

However `/items/contact/+` throws the following message:
`{"errors":[{"message":"You don't have permission to access this.","extensions":{"code":"FORBIDDEN"}}]}`

**Reply by connorwinston** [reply]
The Admin App has no public access so that is not possible, however, public forms is something that is being discussed!
https://github.com/directus/directus/discussions/2977

**Reply by Pachat** [reply]
That's good to know.
I'm just curious to know how those who have Directus apps in production manage users' registration.
Do they do it by hand as admin?

**Reply by connorwinston** [reply]
Most users either invite users using the Admin App, set up SSO, or do it manually by hand. 

Reference this discussion about Public User Registration: https://github.com/directus/directus/discussions/3083
